Assets/Scripts/Manager/ThemeMusicManager.cs

```diff
@@ -44,26 +53,40 @@ public class ThemeMusicManager : MusicManager {
 	public void playThemeSong (string currentLevel)
```

Comment:
too many if statements here, I wouldn't suggest using setSongSwitch here as that requires you to know what the next song going to play
I suggest just check at the beginning, if it's title scene, play track, else fade out the clip. therefore you don't care what the next scene is going to be, you're going to fade it out anyway.